src/components/title-block/TitleBlock.tsx

```diff
@@ -25,11 +25,13 @@ const TitleBlock: FC<TitleBlockProps> = ({ img, translationKey, children }) => {
         <TitleWithDescription
           description={t(`${translationKey}.description`)}
           style={styles.titleWithDescription}
-          title={t(`${translationKey}.title.${userRole}`)}
+          title={t(`${translationKey}.title`)}
         />
         {children && <Box sx={styles.form}>{children}</Box>}
       </Box>
-      {!isMobile && <Box alt='icon' component='img' src={img}></Box>}
+      {!isTablet && !isMobile && (
+        <Box alt='icon' component='img' src={img}></Box>
+      )}
```

Comment:
Why do you do two check if you can do only one?
(isDesctop) instead of (!isTablet && !isMobile_

Reply:
I'll fix it

Reply:
if u change this it will change appearance on categories, subjects, and find-offers where the icon should be hidden only on mobile